spec/plugin.spec.js

```diff
@@ -407,9 +411,7 @@ describe('ManifestPlugin', function() {
           'main.js': 'main.js'
         });
 
-        expect(JSON.parse(stats.compilation.assets['manifest.json'].source())).toEqual({
-          'main.js': 'main.js'
-        });
+        expect(emittedAsset(stats.compilation, 'manifest.json')).toEqual(true)
```

Comment:
Could you check it's content?

Reply:
It looks like Webpack 5 no longer puts the whole file in this object: https://github.com/webpack/changelog-v5/blob/master/README.md#sizeonlysource-after-emit

I could make this test specific to Webpack <= 4 if you want to still verify the content is available here for older versions?

Reply:
NP, I'm planning on updating the tests once v3 is out to rely less on webpack internal